### PR TITLE
feat: skills matrix application

### PR DESCRIPTION
**Pull request by bitpshr**
This pull request implements a skills matrix application. The code could use a bit of a bath, but all functionality is there and should be working as expected.

**Demo URL:**

https://skills-matrix-seven.now.sh/

**Example single-hash URL:**

https://skills-matrix-seven.now.sh/#skills/Paul+Bouchon-ZeOo-VVVVVVVVVQAVVVVQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA

**Example multiple-hash URL:**

https://skills-matrix-seven.now.sh/#compare/Ant+Gubler-ZeOo-VVVVVVVVVQAVVVVQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA,Matt+Gadd-ZeOo-VVVVVVVVVQAVVVVQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA,Paul+Bouchon-ZeOo-VVVVVVVVVQAVVVVQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA,Tom+Dye-ZeOo-VVVVVVVVVQAVVVVQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA

**Preview:**

home

single

multiple

### CODE REVIEW

**Comment by agubler** [comment]
@msssk This example will need to be moved to the new structure that was introduced in #406, there are instructions on the readme (https://github.com/dojo/examples/#adding-an-example, please update them if you find any issues).

Also all the dojo dependencies should now be using the latest release, i.e. `~7.0.0`

**Comment by agubler** [comment]
@msssk I think as part of upgrading to the released version of Dojo there will be a number of changes required, particularly around the use of resources.

File: packages/skills-matrix/src/widgets/skillset-filter/SkillsetFilter.tsx [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+						root: [css.listItemRoot],
+						selected: [css.listItemSelected]
+					},
+					'@dojo/widgets/text-input': {
+						input: [css.input]
+					},
+					'@dojo/widgets/typeahead': {
+						menuWrapper: [css.menu]
+					}
+				}}
```
Comment (tomdye) [inline packages/skills-matrix/src/widgets/skillset-filter/SkillsetFilter.tsx:63]:
```suggestion
				classes={{
					'@dojo/widgets/typeahead': {
						menuWrapper: [css.menu],
						inputInput: [css.input],
						menuItemSelected: [css.listItemSelected]
					}
				}}
```

**Review by rorticus — APPROVED** [review]

**Comment by bitpshr** [comment]
For the sake of a clean review, closing this PR in favor of #410.

**Pull request closed without merging**